src/app/utils.ts

```diff
@@ -27,6 +28,9 @@ export const actionHook = <C extends AnyActionCreator>(actionCreator: C, body: A
 
       if (matches(action)) {
         const catchError = (e: Error) => {
```

Comment:
could be worth extracting this `catchError` function just to reduce complexity in this function